### PR TITLE
NEPT-2073: Add drush command to check iframe usage.

### PR DESCRIPTION
**Pull request by ku-enza**
## Changes
Added a drush command to  be run on a subsite to know how much  iframes are used.

## Command
drush chkif

### CODE REVIEW

**Review by AlbaParrilla — APPROVED** [review]

**Review by ventzie555 — CHANGES_REQUESTED** [review]
I confirm the issue Alba found. After inserting iframe from av portal:

<p><iframe frameborder="0" height="480" id="videoplayer" scrolling="no" src="https://europa.eu/!bq38Td" webkitallowfullscre="" width="852"></iframe>en="true" mozallowfullscreen="true" allowFullScreen="true"&gt;</p>

I got:

bash-4.1$ drush chkif
Checking iframes on site: European Commission

Field Name Field Type NId Node Title AV Portal 
body text_with_summary 2 test NO



Other than that, everything looks good.

**Review by ventzie555 — APPROVED** [review]

**Review by ventzie555 — APPROVED** [review]
Now everything seems to be working good!

File: Check_iframes/check_iframes.drush.inc [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        }
+      } 
+    }
+  }
+  return $results;
+}
+
+/**
+* Function that checks if there is an iframe tag and if there is an AVPortal iframe in a string.
+* It will return 0, 1 (iframe), 2 (av-portal iframe), 3 AVportal Old way.
```
Comment (aritomelo) [inline Check_iframes/check_iframes.drush.inc:125]: What is the difference between 0 and 1?

**Comment by aritomelo** [comment]
The solution looks great. so I am giving my +1

**Pull request merged**